code/modules/spells/spell.dm

```diff
@@ -16,9 +16,9 @@
 	var/base_action = /datum/action/spell_action
 	var/mob/living/owner
 
-/obj/effect/proc_holder/Initialize(mob/living/owner)
+/obj/effect/proc_holder/Initialize(mapload, mob/living/new_owner)
 	. = ..()
-	src.owner = owner
+	owner = new_owner
```

Comment:
wouldn't this be better as a weakref? It leaves a reference to a mob without checking for when it is deleted. At the very least, you might want to register a signal on when the owner gets qdeleted so you can remove them as a reference.

Reply:
:pensive: